src/test/java/com/github/tomakehurst/wiremock/verification/diff/DiffTest.java

```diff
@@ -524,4 +524,24 @@ public void includeSchemeIfSpecified() {
             junitStyleDiffMessage(
                 "https\n" + "ANY\n" + "/thing\n", "http\n" + "ANY\n" + "/thing\n")));
   }
+
+  @Test
+  public void handleExceptionGettingExpressionResult() {
+    Diff diff =
+        new Diff(
+            newRequestPattern(ANY, urlEqualTo("/thing"))
+                .withRequestBody(matchingJsonPath("$.accountNum", equalTo("1234")))
+                .build(),
+            mockRequest().url("/thing").body(""));
+
+    assertThat(
+        diff.toString(),
+        is(
+            junitStyleDiffMessage(
+                "ANY\n" + "/thing\n" + "\n" + "$.accountNum [equalTo] 1234",
+                "ANY\n"
+                    + "/thing\n"
+                    + "\n"
+                    + "Warning: JSON path expression '$.accountNum' failed to match document '' because of error 'json string can not be null or empty'")));
```

Comment:
Does this look OK with the ascii table diff renderer?

Reply:
I think so - added test here:
https://github.com/wiremock/wiremock/pull/2339/commits/c956ace0c1a6e674293b954592619270f16c98b9#diff-9f49c014a4dac088bc7f29ce321b2a663878282e1f3c7c1d4fc00fb592a39348